packages/react-composites/review/react-composites.api.md

```diff
@@ -244,8 +245,15 @@ export type CallCompositeProps = {
     fluentTheme?: PartialTheme | Theme;
     callInvitationURL?: string;
     onRenderAvatar?: (props: PlaceholderProps, defaultOnRender: (props: PlaceholderProps) => JSX.Element) => JSX.Element;
+    overrideCallControlButtons?: (defaultButtons: CallControlButtonCollection) => CallControlButtonCollection;
 };
 
+// @public (undocumented)
+export type CallControlButton = DefaultCallControlButton | CustomCallControlButton;
```

Comment:
We could extend this further to allow rendering anything in here with 
```suggestion
export type CallControlButton = DefaultCallControlButton | CustomCallControlButton | JSX.Element
```

Reply:
Naah, I'd recommend against opening that door.